README.md

```diff
@@ -26,24 +26,6 @@ See [ElastiBuild](https://github.com/elastic/elastic-stack-installers/wiki/Elast
 
 **NOTE**: *Building from source should only be done for development purposes. Only the officially distributed and signed Elastic Stack Installers should be used in production. Using unofficial Elastic Stack Installers is not supported.*
 
----
-
-## Bumping version
-
-### After a patch release
-
-Update version in `Directory.Build.props` in the branch for the related minor version (ex: https://github.com/elastic/elastic-stack-installers/pull/183).
```

Comment:
Maybe `Directory.Build.props` should have a place-holder version like `0.0.0` to avoid confusion ?

Reply:
> Maybe `Directory.Build.props` should have a place-holder version like `0.0.0` to avoid confusion ?

Not sure I follow, in this PR we are completely removing the file `Directory.Build.props` as it doesn't seem to be needed anywhere. Additionally all references to manual tasks in the README file.

Do you think we still need to keep the file?

Reply:
We're all good, my bad. I was looking for a change that removes the property not the entire file, I didn't realise it only holds this one value.